Tasks/Common/VstsAzureRestHelpers_/VstsAzureRestHelpers_.psm1

```diff
@@ -100,11 +100,11 @@ function IsAzureRmConnection {
     param([Parameter(Mandatory = $true)] $connectionType)
 
     Write-Verbose "Connection type used is $connectionType"
-    if (($connectionType -eq $spnConnection) -or ($connectionType -eq $MsiConnection)) {
-        return $true
-    }
-    else {
-        return $false
+    switch ($connectionType) {
+        $spnConnection { $true }
+        $MsiConnection { $true }
+        $wifConnection { $true }
+        Default { $false }
```

Comment:
Could you please add some unit tests for this function?

Reply:
Do we really need to test the switch statement?

Reply:
I don't really see the value of this unit test.
I used switch just to clean up a code a bit by simplifying nested if statements.